excel/src/main/java/org/apache/metamodel/excel/ExcelDataContext.java

```diff
@@ -209,13 +205,13 @@ protected void onSchemaCacheRefreshed() {
         return null;
     }
 
-    private SpreadsheetReaderDelegate getSpreadsheetReaderDelegate(Ref<InputStream> inputStream)
+    private SpreadsheetReaderDelegate getSpreadsheetReaderDelegate()
             throws MetaModelException {
         if (_spreadsheetReaderDelegate == null) {
             synchronized (this) {
                 if (_spreadsheetReaderDelegate == null) {
                     try {
-                        if (POIXMLDocument.hasOOXMLHeader(inputStream.get())) {
+                        if (POIXMLDocument.hasOOXMLHeader(getInputStream())) {
```

Comment:
Wait, does that mean that your Java is doing like Travis? We're totally doing this on your computer. If it's your old laptop, bring it! If it's the Mac... I'd consider it a hardware issue :-P